### PR TITLE
ci: use APISIX master branch code to run e2e test

### PR DESCRIPTION
**Pull request by nic-chen**
use APISIX master branch code to run e2e test.

### CODE REVIEW

File: api/test/docker/Dockerfile-apisix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+COPY --from=production-stage /usr/local/apisix/ /usr/local/apisix/
+COPY --from=production-stage /usr/bin/apisix /usr/bin/apisix
+
+ENV PATH=$PATH:/usr/local/openresty/luajit/bin:/usr/local/openresty/nginx/sbin:/usr/local/openresty/bin
+
+EXPOSE 9080 9443
+
+CMD ["sh", "-c", "/usr/bin/apisix init && /usr/bin/apisix init_etcd && /usr/local/openresty/bin/openresty -p /usr/local/apisix -g 'daemon off;'"]
+
+STOPSIGNAL SIGQUIT
```
Comment (juzhiyuan) [inline api/test/docker/Dockerfile-apisix:38]: New line is needed

**Comment by moonming** [comment]
we will run apache release of apisix later, so we need to add ENV for the version of apisix

**Comment by nic-chen** [comment]
OK

**Comment by membphis** [comment]
@nic-chen please rebase your branch, PR https://github.com/apache/apisix-dashboard/pull/867 was merged

**Comment by nic-chen** [comment]
@membphis  updated.

**Review by membphis — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**